spec/controllers/v0/institutions_controller_spec.rb

```diff
@@ -40,15 +40,15 @@
   context 'autocomplete results' do
     it 'returns collection of matches' do
       create(:version, :production)
-      7.times { create(:institution, :contains_harv) }
+      7.times { create(:institution, :contains_harv, approved: true) }
       get :autocomplete, term: 'harv', version: 'production'
       expect(response.content_type).to eq('application/json')
       expect(response).to match_response_schema('autocomplete')
     end
 
     it 'returns empty collection on missing term parameter' do
       create(:version, :production)
-      7.times { create(:institution, :contains_harv) }
+      7.times { create(:institution, :contains_harv, approved: true) }
```

Comment:
Does this need to be specifically seven times, and if so, please comment as to why so we can avoid [mystery guest smell](https://thoughtbot.com/blog/mystery-guest).

Reply:
Also to reduce time it takes - we're hitting the DB 7 times to create 7 records, this is a lot of overhead for one example.

Reply:
`approved` is a new column (via the most recent previous PR) that defaults to `false` at the database level.
these controller methods only return results where `approved: true`, so it needs to be specified here, or he can change the factory to specify `approved: true`.  I am unsure if that would affect other existing specs.

Reply:
Not sure why the 7.times is used there, I just added the approved: true portion to that line.